language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/inlayhint/InlayHintProvider.java

```diff
@@ -104,8 +131,9 @@ public static List<InlayHint> getInlayHint(InlayHintContext context) {
                         inlayHints.add(inlayHint);
                     }
                     break;
-                } else if (parameterSymbols.getLeft().get(argumentIndex).getName().isEmpty()) {
-                    break;
+                } else if (parameterSymbols.getLeft().get(argumentIndex).getName().isEmpty()
+                        || parameterSymbols.getLeft().get(argumentIndex).getName().get().startsWith("$")) {
+                    continue;
```

Comment:
Let's not show anything when there are syntax errors in the invokeable.Let's add a test case for this as well

Reply:
Done